### ISSUE TITLE
Can't run motion: Inconsistency detected

### ISSUE THREAD

**Issue by fraschizzato** [Motion-Project/motion#1186]
0. Reviewed guide and contributing documents? (Yes/No): Yes
1. version [x.y.z, hash, other]:  4.1.1-1
2. installed as a package or compiled from sources [deb, rpm, git, other]: deb from apt 
3. standalone or part of third party [motion, MotionEyeOS, other]:  standalone
4. video stream source [V4L (card or USB), net cam (mjpeg, rtsp, other), mmal]: USB 
5. hardware [x86, ARM, other]:  ARM
6. operating system [16.04, Stretch, etc, FreeBSD, other]: Raspbian Buster

 Motion was installed months ago and was working fine. Now, after a reboot, I've found that it won't start:
As service i get "active (exited)" and it's not working.
Running via command line i get 
```
Inconsistency detected by ld.so: dl-lookup.c: 948: _dl_setup_hash: Assertion `(bitmask_nwords & (bitmask_nwords - 1)) == 0' failed!
```
I would like to recover that wihout reflash everything, what could be the Issue? 
Tried to reinstall motion, by reinstall flag, by remove and install and by autoremove and purge

**Comment by genius3000** [comment]
This looks like a problem with a shared library. A little googling has revealed some suggestions:
[https://stackoverflow.com/questions/22564780/debugging-ld-inconsistency-detected-by-ld-so](https://stackoverflow.com/questions/22564780/debugging-ld-inconsistency-detected-by-ld-so) which suggests setting the environment variable of `LD_DEBUG=all`. A little more information on that is available [here](http://www.bnikolic.co.uk/blog/linux-ld-debug.html).
Try running `LD_DEBUG=all motion` and see which library it is failing on.

Another suggestion is using `strace`: [https://www.linuxquestions.org/questions/slackware-14/how-to-fix-inconsistency-detected-by-ld-so-4175541607/](https://www.linuxquestions.org/questions/slackware-14/how-to-fix-inconsistency-detected-by-ld-so-4175541607/).

If you can narrow down the library or libraries, try reinstalling them. I'm assuming this is on an RPi and it might not hurt to check on your SD card (or whatever they have for storage now), perhaps it is starting to fall.

**Comment by fraschizzato** [comment]
Yes, thanks. Found that first issues can be fixed by `sudo apt install openjdk-8-jdk`  and `sudo update-alternatives --config java`
But then another error, so I had to reinstall libflac8...
Then a segmentation fault.... 

At this point is better to to replace the sdcard with a previous backup...
